templates/AureliaSpa/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+var isDevBuild = process.argv.indexOf('--env.prod') < 0;
+var path = require('path');
+var webpack = require('webpack');
+var AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+var srcDir = path.resolve('./ClientApp');
+var rootDir = path.resolve();
+var outDir = path.resolve('./wwwroot/dist');
+var baseUrl = '/';
+var project = require('./package.json');
+var aureliaModules = Object.keys(project.dependencies).filter(dep => dep.startsWith('aurelia-'));
+
+// Configuration for client-side bundle suitable for running in browsers
+var clientBundleConfig = {
+     resolve: { extensions: [ '.js', '.ts' ] },
+     devtool: isDevBuild ? 'inline-source-map' : null,
+     entry: {
+        'app': [], // <-- this array will be filled by the aurelia-webpack-plugin
```

Comment:
Latest Webpack@beta currently errors when an array is empty (it's an ongoing bug). I might be better to instead point it to where the `main.js` file is, as that should be in the context anyway.